desci-server/test/integration/research-assistant/researchAssistantMetering.test.ts

```
@@ -0,0 +1,789 @@
+import request from 'supertest';
+import { addMonths, subMonths } from 'date-fns';
+
+import { prisma } from '../../../src/client.js';
+import { server } from '../../../src/server.js';
+import { FeatureLimitsService } from '../../../src/services/FeatureLimits/FeatureLimitsService.js';
+import { FeatureUsageService, LimitExceededError } from '../../../src/services/FeatureLimits/FeatureUsageService.js';
+
+server.ready().then((_) => {
+  console.log('server is ready');
+});
+export const app = server.app;
```

Biome (2.1.2)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)